templates/lxc-download.in

```diff
@@ -126,10 +126,16 @@ gpg_setup() {
     return
   fi
 
+  
+
   echo "Setting up the GPG keyring"
 
   mkdir -p "${DOWNLOAD_TEMP}/gpg"
   chmod 700 "${DOWNLOAD_TEMP}/gpg"
+
+  if [ --standard-resolver "${STANDARD_RESOLVER:-}" ]; then
```

Comment:
This is invalid shell, you should check for something like `"${DOWNLOAD_STANDARD_RESOLVER}" = "true"` and make sure to initialize `DOWNLOAD_STANDARD_RESOLVER` to `false` at the top of the file.